### ISSUE TITLE
Addd logging of user input of the role and goals

### ISSUE THREAD

**Issue by dsheshin** [Significant-Gravitas/Auto-GPT#2428]
### Duplicates

- [X] I have searched the existing issues

### Summary 💡

Now logs reflect only gpt's response but i don't really remember what exactly i input before.  Please log it same as in the console. 
Current logging makes it a lot harder to debug

### Examples 🌈

All packages are installed.
Welcome back!  Would you like me to return to being sc3?
Continue with the last settings?
Name:  sc3
Role:  warhammer 40k writer
Goals: ['research the theme', 'do a 5000 symbols structurized explanation on wh40k lore', 'terminate']
Continue (y/n): n
Welcome to Auto-GPT!  run with '--help' for more information.
Create an AI-Assistant:  Enter the name of your AI and its role below. Entering nothing will load defaults.
Name your AI:  For example, 'Entrepreneur-GPT'
AI Name: da23eads
da23eads here!  I am at your service.
Describe your AI's role:  For example, 'an AI designed to autonomously develop and run businesses with the sole goal of increasing your net worth.'
da23eads is: wh 40k writer
Enter up to 5 goals for your AI:  For example: Increase net worth, Grow Twitter Account, Develop and manage multiple businesses autonomously'
Enter nothing to load defaults, enter nothing when finished.
Goal 1: research the theme
Goal 2: do a plot esplanation on warhammer 40k universe
Goal 3: terminate
Goal 4:
Using memory of type:  LocalCache
Using Browser:  chrome
- Thinking...

### Motivation 🔦

make the world better

**Comment by ntindle** [comment]
Are you using pinecone?

**Comment by Grindin124** [comment]
No, I have not heard of that. I did get it figured out, it was an issue with API. New to this, and trying to get a handle on it. What is Pinecone, and what benefits does it provide?

**Comment by Pwuts** [comment]
For this we have the `--debug` flag. Closing as resolved.

**Comment by dsheshin** [comment]
> --debug

this is overkill as for general analysis purposes there is no need for real debugging. just save my input as default that's it

**Comment by Pwuts** [comment]
Your input is saved in `ai_settings.yml`
